packages/UsageAnalysis/files/Test Track/Viewers/Filter panel/Filter with hidden columns.md

```diff
@@ -0,0 +1,24 @@
+### Verify common Hidden Columns Behavior in the Filter Panel
+Ensure that columns are correctly shown or hidden in the filter panel based on their visibility settings.
+* Preconditions: SPGI dataset is loaded into the platform.
```

Comment:
Open dataset using Samples dataset icon is better here to give a hint hoe to do it

Reply:
Agree. Changed